microsoftgraph/microsoft-graph-explorer-v4
# Fix: Permissions scrollbar

## Overview
Closes #2563 

### Demo
<img width="679" alt="Screenshot 2023-05-16 at 12 18 54 PM" src="https://github.com/microsoftgraph/microsoft-graph-explorer-v4/assets/45680252/69cee826-b9bf-48c0-af94-967d9144b51a">


### Notes
The DetailsList component has a wrapper div that is added at runtime with a classname of ms-ViewPort. So, we have to add css rules to the div dynamically

## Testing Instructions

* How to test this PR
1. Resize the screen so that the app is in mobile view. 
2. Type the query: https://graph.microsoft.com/v1.0/me/photo on the query textbox
3. Click on the Modify Permissions tab
4. Notice that the scrollbar that appears is at the bottom of the tab
